test/scenarios.spec.ts

```diff
@@ -370,7 +370,10 @@ suite('== Custom List APL Props ==', () => {
                         },
                     ],
                 },
-                valueRenderer: (x: string, input) => `Wizard House: ${x}`,
+                valueRenderer: (x: string, input) => ({
```

Comment:
please have a think if there are test cases re: component mode would be valuable.  In particular things to do with the processing rather than the APL document result such as automatic defaults, handling of undefined, a minimal example as a test case for documentation purposes and so on.

Reply:
Yes, will add more specific test cases/scenarios in new PR to address this new feature(APL Component Mode)